game/src/sandbox/dashboards/travel_times.rs

```diff
@@ -13,6 +13,7 @@ use widgetry::{
     Panel, State, Text, TextExt, Toggle, Widget,
 };
 
+use super::trip_problems::{problem_matrix, ProblemType, TripProblemFilter};
```

Comment:
This entire dash now scrolls very slowly in release mode for me. The risk exposure one doesn't, so I suspect the `CompareTimes` widget is to blame. I can dig in this weekend if you'd like. I'm really quickly trying disabling https://github.com/a-b-street/abstreet/blob/446a21696dd60d8c15a5d3614989df1d4d9d661b/widgetry/src/widgets/compare_times.rs#L148 to see if we need to be more clever here...

Reply:
That doesn't help. Not a blocker right now, I can dig in tomorrow.

Reply:
I'm digging into this perf bit now...

My screen is tall enough that scrolling isn't induced at all, so I forgot to test this.